src/tools/fuzzing/fuzzing.cpp

```diff
@@ -2148,14 +2148,17 @@ Expression* TranslateToFuzzReader::_makeConcrete(Type type) {
     options.add(FeatureSet::ReferenceTypes, &Self::makeRefIsNull);
     options.add(FeatureSet::ReferenceTypes | FeatureSet::GC,
                 &Self::makeRefEq,
-                &Self::makeRefTest,
                 &Self::makeI31Get);
     options.add(FeatureSet::ReferenceTypes | FeatureSet::GC |
                   FeatureSet::Strings,
                 &Self::makeStringEncode,
                 &Self::makeStringEq,
                 &Self::makeStringMeasure,
                 &Self::makeStringGet);
+    if (type.isInexact() || wasm.features.hasCustomDescriptors()) {
```

Comment:
Maybe worth a comment here?

Reply:
Actually, this isn't even necessary because we know type is `i32` here.